tests/foundry/foundry.k.check.expected

```diff
@@ -1,9 +1,11 @@
 requires "edsl.md"
+requires "foundry.md"
 requires "lemmas/lemmas.k"
 requires "lemmas/int-simplification.k"
 
 module ADDRTEST-BIN-RUNTIME
     imports public EDSL
+    imports public FOUNDRY
     imports public LEMMAS
     imports public INT-SIMPLIFICATION
```

Comment:
I guess `FOUNDRY` already imports `EDSL`? So we can remove the `EDSL` import from these modules now?

Reply:
That's correct. I'm updating that now.

Reply:
@ehildenb  `FOUNDRY` currently imports only the `BIN-RUNTIME` module from `EDSL` (see line 83 from `foundry.md` above). I could just import the entire `EDSL` module in `FOUNDRY` and remove `EDSL` import here.